Makefile

```diff
@@ -46,18 +46,23 @@ installtools: ## Install development related tools
 	go get github.com/kardianos/govendor
 	go get github.com/maxbrunsfeld/counterfeiter
 	go get github.com/yvasiyarov/swagger
+	go get github.com/rakyll/statik
 
 build: clean build/linux build/darwin ## Build for linux and darwin (save to OUTPUT_DIR/BIN)
 
-build/linux: clean/linux ## Build for linux (save to OUTPUT_DIR/BIN)
+build/linux: clean/linux build/ui ## Build for linux (save to OUTPUT_DIR/BIN)
 	GOOS=linux go build -a -installsuffix cgo -ldflags "-X main.version=$(RELEASE_VER)" -o $(OUTPUT_DIR)/$(BIN)-linux .
 
-build/darwin: clean/darwin ## Build for darwin (save to OUTPUT_DIR/BIN)
+build/darwin: clean/darwin build/ui ## Build for darwin (save to OUTPUT_DIR/BIN)
 	GOOS=darwin go build -a -installsuffix cgo -ldflags "-X main.version=$(RELEASE_VER)" -o $(OUTPUT_DIR)/$(BIN)-darwin .
 
 build/docs: ## Build markdown docs from swagger comments
 	swagger -apiPackage="github.com/9corp/9volt" -format=markdown -output=docs/api/README.md
 
+build/ui: ui
```

Comment:
This will build the UI and compile it into the `statik.go` file so that it's fully browsable from a single executable.